src/Query/Builder.php

```diff
@@ -114,12 +114,23 @@ class Builder extends BaseBuilder
      */
     protected $conversion = [
         '=' => '=',
```

Comment:
What purpose does it serve to have `=` map to itself? This seems to be the only thing leftover from the original `$conversion` map.